README.md

```diff
@@ -1,5 +1,13 @@
-Нужен Python, PostgreSQL.
+Python 2, PostgreSQL.
 
-    python -m pip install virtualenv
-    python -m virtualenv .v
-    .v/bin/pip install -r requirements.txt
+```
+# Installation on Fedora requires these dependencies
+sudo dnf install python2-devel GeoIP-devel libpq-devel
+
+# Not compatible with Python 3
+pipenv --python 2
+pipenv install
+
+# Run server
+pipenv run python index.py
+```
```

Comment:
Thanks for this update! Does `pipenv` required to be preinstalled to use it? I wondered if some more boilerplate approach as `python -m venv` can be used (unfortunatly my example works only for py3)?

Reply:
`python3 -m venv` or `pypy3 -m venv`

Reply:
@cclauss this project is not yet Python 3 ready.

@tbicr yes, `pipenv` needs to be preinstalled, much like `virtualenv`. I prefer `pipenv`, because it handles installation automatically and allows to just logout from its shell instead of deactivation etc.

Reply:
I know.  #11 builds on legacy Python.